constraints/src/property.rs

```diff
@@ -0,0 +1,168 @@
+//! Constants identifying the properties of a [`MediaStreamTrack`][media_stream_track] object,
```

Comment:
Would it be valuable to wrap these in a new type to prevent mistakes with regular string values?

Reply:
Absolutely! Fixed, by introducing `pub struct MediaTrackProperty(Cow<'static, str>);`.

This way we still allow working with unanticipated future property names via either `MediaTrackProperty::from("foo")`, `MediaTrackProperty::from("foo".to_owned())`, or the non-allocating `MediaTrackProperty::named("foo")`.